.github/workflows/common-docker.yml

```diff
@@ -0,0 +1,107 @@
+name: Docker
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        description: Environment (network) for workflow execution
+        required: false
+        type: string
+      download-artifact-name:
+        required: true
+        type: string
+      download-artifact-path:
+        required: true
+        type: string
+      download-artifact-tar:
+        description: Set to true if provided `download-artifact-path` is compressed tar file
+        required: true
+        default: false
+        type: boolean
+      image-name:
+        description: Docker image name
+        required: true
+        type: string
+      image-push:
+        required: true
+        default: false
+        type: boolean
+      image-context:
```

Comment:
Could we add `description` here? So that it would be clear that context refers to the path to the folder with the files that should be built